[
{
	"type": "CONFLUENCE",
	"label": "CF conflict",
	"body": {
		"description": "SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,$stack11 := @caughtexception,60,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,stmt#2 = $stack11,62,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,specialinvoke tmp$1952194564.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(stmt#2),64,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, specialinvoke tmp$1952194564.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(stmt#2), 64)) SINK=>BASE: Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,$stack11 := @caughtexception,60,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,stmt#2 = $stack11,62,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,specialinvoke tmp$1040493509.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(stmt#2),64,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, specialinvoke tmp$1040493509.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(stmt#2), 64))",
		"interference": [
			{
				"type": "source1",
				"branch": "L",
				"text": "$stack11 := @caughtexception",
				"location": {
					"file": "",
					"class": "org.jdbi.jdbi3.Handle",
					"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
					"line": 60
				},
				"stackTrace": [{
	"class": "org.jdbi.jdbi3.Handle",
	"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
	"line": 60
},{
	"class": "org.jdbi.jdbi3.Handle",
	"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
	"line": 62
},{
	"class": "org.jdbi.jdbi3.Handle",
	"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
	"line": 64
}]
			},
			{
				"type": "source2",
				"branch": "R",
				"text": "$stack11 := @caughtexception",
				"location": {
					"file": "",
					"class": "org.jdbi.jdbi3.Handle",
					"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
					"line": 60
				},
				"stackTrace": [{
	"class": "org.jdbi.jdbi3.Handle",
	"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
	"line": 60
},{
	"class": "org.jdbi.jdbi3.Handle",
	"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
	"line": 62
},{
	"class": "org.jdbi.jdbi3.Handle",
	"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
	"line": 64
}]
			},
			{
				"type": "confluence",
				"branch": "B",
				"text": "specialinvoke tmp$1952194564.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(stmt#2)",
				"location": {
					"file": "",
					"class": "org.jdbi.jdbi3.Handle",
					"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
					"line": 64
				}
			}
		]
	}
},
{
	"type": "CONFLUENCE",
	"label": "CF conflict",
	"body": {
		"description": "SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack10 = new org.jdbi.jdbi3.JDBIException,55,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,this := @this: org.jdbi.jdbi3.JDBIException,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l0 := @this: java.lang.RuntimeException,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1),96,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) SINK=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack9 := @caughtexception,45,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1),96,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96))",
		"interference": [
			{
				"type": "source1",
				"branch": "L",
				"text": "$stack9 := @caughtexception",
				"location": {
					"file": "",
					"class": "org.jdbi.jdbi3.Handle$1",
					"method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
					"line": 45
				},
				"stackTrace": [{
	"class": "org.jdbi.jdbi3.Handle$1",
	"method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
	"line": 45
},{
	"class": "org.jdbi.jdbi3.Handle$1",
	"method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
	"line": 53
},{
	"class": "org.jdbi.jdbi3.JDBIException",
	"method": "<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.RuntimeException",
	"method": "<java.lang.RuntimeException: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.RuntimeException",
	"method": "<java.lang.RuntimeException: void <init>(java.lang.Throwable)>",
	"line": 96
}]
			},
			{
				"type": "source2",
				"branch": "R",
				"text": "$stack10 = new org.jdbi.jdbi3.JDBIException",
				"location": {
					"file": "",
					"class": "org.jdbi.jdbi3.Handle$1",
					"method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
					"line": 55
				},
				"stackTrace": [{
	"class": "org.jdbi.jdbi3.Handle$1",
	"method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
	"line": 55
},{
	"class": "org.jdbi.jdbi3.JDBIException",
	"method": "<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.RuntimeException",
	"method": "<java.lang.RuntimeException: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.RuntimeException",
	"method": "<java.lang.RuntimeException: void <init>(java.lang.Throwable)>",
	"line": 96
}]
			},
			{
				"type": "confluence",
				"branch": "B",
				"text": "specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1)",
				"location": {
					"file": "",
					"class": "java.lang.RuntimeException",
					"method": "<java.lang.RuntimeException: void <init>(java.lang.Throwable)>",
					"line": 96
				}
			}
		]
	}
},
{
	"type": "CONFLUENCE",
	"label": "CF conflict",
	"body": {
		"description": "SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,$stack11 := @caughtexception,60,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,stmt#2 = $stack11,62,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1),102,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102)) SINK=>BASE: Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,$stack11 := @caughtexception,60,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,stmt#2 = $stack11,62,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1),102,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102))",
		"interference": [
			{
				"type": "source1",
				"branch": "L",
				"text": "$stack11 := @caughtexception",
				"location": {
					"file": "",
					"class": "org.jdbi.jdbi3.Handle",
					"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
					"line": 60
				},
				"stackTrace": [{
	"class": "org.jdbi.jdbi3.Handle",
	"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
	"line": 60
},{
	"class": "org.jdbi.jdbi3.Handle",
	"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
	"line": 62
},{
	"class": "org.jdbi.jdbi3.JDBIException",
	"method": "<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.RuntimeException",
	"method": "<java.lang.RuntimeException: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.Exception",
	"method": "<java.lang.Exception: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.Exception",
	"method": "<java.lang.Exception: void <init>(java.lang.Throwable)>",
	"line": 102
}]
			},
			{
				"type": "source2",
				"branch": "R",
				"text": "$stack11 := @caughtexception",
				"location": {
					"file": "",
					"class": "org.jdbi.jdbi3.Handle",
					"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
					"line": 60
				},
				"stackTrace": [{
	"class": "org.jdbi.jdbi3.Handle",
	"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
	"line": 60
},{
	"class": "org.jdbi.jdbi3.Handle",
	"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
	"line": 62
},{
	"class": "org.jdbi.jdbi3.JDBIException",
	"method": "<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.RuntimeException",
	"method": "<java.lang.RuntimeException: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.Exception",
	"method": "<java.lang.Exception: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.Exception",
	"method": "<java.lang.Exception: void <init>(java.lang.Throwable)>",
	"line": 102
}]
			},
			{
				"type": "confluence",
				"branch": "B",
				"text": "specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1)",
				"location": {
					"file": "",
					"class": "java.lang.Exception",
					"method": "<java.lang.Exception: void <init>(java.lang.Throwable)>",
					"line": 102
				}
			}
		]
	}
},
{
	"type": "CONFLUENCE",
	"label": "CF conflict",
	"body": {
		"description": "SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack9 := @caughtexception,45,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,$stack5 = virtualinvoke l1.<java.lang.Throwable: java.lang.String toString()>(),311,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102) => (<java.lang.Throwable: void <init>(java.lang.Throwable)>, $stack5 = virtualinvoke l1.<java.lang.Throwable: java.lang.String toString()>(), 311)) SINK=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,$stack5 = virtualinvoke l1.<java.lang.Throwable: java.lang.String toString()>(),311,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102) => (<java.lang.Throwable: void <init>(java.lang.Throwable)>, $stack5 = virtualinvoke l1.<java.lang.Throwable: java.lang.String toString()>(), 311))",
		"interference": [
			{
				"type": "source1",
				"branch": "L",
				"text": "e = $stack9",
				"location": {
					"file": "",
					"class": "org.jdbi.jdbi3.Handle$1",
					"method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
					"line": 53
				},
				"stackTrace": [{
	"class": "org.jdbi.jdbi3.Handle$1",
	"method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
	"line": 53
},{
	"class": "org.jdbi.jdbi3.JDBIException",
	"method": "<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.RuntimeException",
	"method": "<java.lang.RuntimeException: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.Exception",
	"method": "<java.lang.Exception: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.Throwable",
	"method": "<java.lang.Throwable: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.Throwable",
	"method": "<java.lang.Throwable: void <init>(java.lang.Throwable)>",
	"line": 311
}]
			},
			{
				"type": "source2",
				"branch": "R",
				"text": "$stack9 := @caughtexception",
				"location": {
					"file": "",
					"class": "org.jdbi.jdbi3.Handle$1",
					"method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
					"line": 45
				},
				"stackTrace": [{
	"class": "org.jdbi.jdbi3.Handle$1",
	"method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
	"line": 45
},{
	"class": "org.jdbi.jdbi3.Handle$1",
	"method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
	"line": 53
},{
	"class": "org.jdbi.jdbi3.JDBIException",
	"method": "<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.RuntimeException",
	"method": "<java.lang.RuntimeException: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.Exception",
	"method": "<java.lang.Exception: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.Throwable",
	"method": "<java.lang.Throwable: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.Throwable",
	"method": "<java.lang.Throwable: void <init>(java.lang.Throwable)>",
	"line": 311
}]
			},
			{
				"type": "confluence",
				"branch": "B",
				"text": "$stack5 = virtualinvoke l1.<java.lang.Throwable: java.lang.String toString()>()",
				"location": {
					"file": "",
					"class": "java.lang.Throwable",
					"method": "<java.lang.Throwable: void <init>(java.lang.Throwable)>",
					"line": 311
				}
			}
		]
	}
},
{
	"type": "CONFLUENCE",
	"label": "CF conflict",
	"body": {
		"description": "SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,$stack10 = new org.jdbi.jdbi3.JDBIException,55,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,this := @this: org.jdbi.jdbi3.JDBIException,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l0 := @this: java.lang.RuntimeException,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l0 := @this: java.lang.Exception,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l0 := @this: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l0.<java.lang.Throwable: java.lang.Throwable cause> = l1,312,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102) => (<java.lang.Throwable: void <init>(java.lang.Throwable)>, l0.<java.lang.Throwable: java.lang.Throwable cause> = l1, 312)) SINK=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<java.lang.RuntimeException: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) => Node(<java.lang.Exception: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l1 := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102)) => Node(<java.lang.Throwable: void <init>(java.lang.Throwable)>,l0.<java.lang.Throwable: java.lang.Throwable cause> = l1,312,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7) => (<java.lang.RuntimeException: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Exception: void <init>(java.lang.Throwable)>(l1), 96) => (<java.lang.Exception: void <init>(java.lang.Throwable)>, specialinvoke l0.<java.lang.Throwable: void <init>(java.lang.Throwable)>(l1), 102) => (<java.lang.Throwable: void <init>(java.lang.Throwable)>, l0.<java.lang.Throwable: java.lang.Throwable cause> = l1, 312))",
		"interference": [
			{
				"type": "source1",
				"branch": "L",
				"text": "e = $stack9",
				"location": {
					"file": "",
					"class": "org.jdbi.jdbi3.Handle$1",
					"method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
					"line": 53
				},
				"stackTrace": [{
	"class": "org.jdbi.jdbi3.Handle$1",
	"method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
	"line": 53
},{
	"class": "org.jdbi.jdbi3.JDBIException",
	"method": "<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.RuntimeException",
	"method": "<java.lang.RuntimeException: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.Exception",
	"method": "<java.lang.Exception: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.Throwable",
	"method": "<java.lang.Throwable: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.Throwable",
	"method": "<java.lang.Throwable: void <init>(java.lang.Throwable)>",
	"line": 312
}]
			},
			{
				"type": "source2",
				"branch": "R",
				"text": "$stack10 = new org.jdbi.jdbi3.JDBIException",
				"location": {
					"file": "",
					"class": "org.jdbi.jdbi3.Handle$1",
					"method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
					"line": 55
				},
				"stackTrace": [{
	"class": "org.jdbi.jdbi3.Handle$1",
	"method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
	"line": 55
},{
	"class": "org.jdbi.jdbi3.JDBIException",
	"method": "<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.RuntimeException",
	"method": "<java.lang.RuntimeException: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.Exception",
	"method": "<java.lang.Exception: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.Throwable",
	"method": "<java.lang.Throwable: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "java.lang.Throwable",
	"method": "<java.lang.Throwable: void <init>(java.lang.Throwable)>",
	"line": 312
}]
			},
			{
				"type": "confluence",
				"branch": "B",
				"text": "l0.<java.lang.Throwable: java.lang.Throwable cause> = l1",
				"location": {
					"file": "",
					"class": "java.lang.Throwable",
					"method": "<java.lang.Throwable: void <init>(java.lang.Throwable)>",
					"line": 312
				}
			}
		]
	}
},
{
	"type": "CONFLUENCE",
	"label": "CF conflict",
	"body": {
		"description": "SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,$stack11 := @caughtexception,60,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,stmt#2 = $stack11,62,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) SINK=>BASE: Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,$stack11 := @caughtexception,60,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,stmt#2 = $stack11,62,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62))",
		"interference": [
			{
				"type": "source1",
				"branch": "L",
				"text": "$stack11 := @caughtexception",
				"location": {
					"file": "",
					"class": "org.jdbi.jdbi3.Handle",
					"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
					"line": 60
				},
				"stackTrace": [{
	"class": "org.jdbi.jdbi3.Handle",
	"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
	"line": 60
},{
	"class": "org.jdbi.jdbi3.Handle",
	"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
	"line": 62
}]
			},
			{
				"type": "source2",
				"branch": "R",
				"text": "$stack11 := @caughtexception",
				"location": {
					"file": "",
					"class": "org.jdbi.jdbi3.Handle",
					"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
					"line": 60
				},
				"stackTrace": [{
	"class": "org.jdbi.jdbi3.Handle",
	"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
	"line": 60
},{
	"class": "org.jdbi.jdbi3.Handle",
	"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
	"line": 62
}]
			},
			{
				"type": "confluence",
				"branch": "B",
				"text": "stmt#2 = $stack11",
				"location": {
					"file": "",
					"class": "org.jdbi.jdbi3.Handle",
					"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
					"line": 62
				}
			}
		]
	}
},
{
	"type": "CONFLUENCE",
	"label": "CF conflict",
	"body": {
		"description": "SOURCE=>BASE: Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,$stack11 := @caughtexception,60,SourceNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>,stmt#2 = $stack11,62,SinkNode, path: (<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause),7,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)) SINK=>BASE: Node(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>,e = $stack9,53,SourceNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,cause := @parameter0: java.lang.Throwable,-1,SimpleNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)) => Node(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>,specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause),7,SinkNode, path: (<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55) => (<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7))",
		"interference": [
			{
				"type": "source1",
				"branch": "L",
				"text": "e = $stack9",
				"location": {
					"file": "",
					"class": "org.jdbi.jdbi3.Handle$1",
					"method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
					"line": 53
				},
				"stackTrace": [{
	"class": "org.jdbi.jdbi3.Handle$1",
	"method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
	"line": 53
},{
	"class": "org.jdbi.jdbi3.JDBIException",
	"method": "<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "org.jdbi.jdbi3.JDBIException",
	"method": "<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>",
	"line": 7
}]
			},
			{
				"type": "source2",
				"branch": "R",
				"text": "$stack11 := @caughtexception",
				"location": {
					"file": "",
					"class": "org.jdbi.jdbi3.Handle",
					"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
					"line": 60
				},
				"stackTrace": [{
	"class": "org.jdbi.jdbi3.Handle",
	"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
	"line": 60
},{
	"class": "org.jdbi.jdbi3.Handle",
	"method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
	"line": 62
},{
	"class": "org.jdbi.jdbi3.JDBIException",
	"method": "<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>",
	"line": -1
},{
	"class": "org.jdbi.jdbi3.JDBIException",
	"method": "<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>",
	"line": 7
}]
			},
			{
				"type": "confluence",
				"branch": "B",
				"text": "specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause)",
				"location": {
					"file": "",
					"class": "org.jdbi.jdbi3.JDBIException",
					"method": "<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>",
					"line": 7
				}
			}
		]
	}
}

,

{
	"type": "OAINTER",
	"label": "OA conflict",
	"body": {
		"description": "this.<org.jdbi.jdbi3.ResultSetRow: java.sql.ResultSet rs> - this.<org.jdbi.jdbi3.ResultSetRow: java.sql.ResultSet rs>",
		"interference": [
			{
				"type": "declaration",
				"branch": "L",
				"text": "this.<org.jdbi.jdbi3.ResultSetRow: java.sql.ResultSet rs> = rs",
				"location": {
					"file": "",
					"class": "org.jdbi.jdbi3.ResultSetRow",
					"method": "<init>",
					"line": 12
				},
				"stackTrace": [{
	"class": "org.jdbi.jdbi3.Handle$1",
	"method": "computeNext",
	"line": 45
},{
	"class": "org.jdbi.jdbi3.ResultSetRow",
	"method": "<init>",
	"line": 12
}]
			},
			{
				"type": "override",
				"branch": "R",
				"text": "this.<org.jdbi.jdbi3.ResultSetRow: java.sql.ResultSet rs> = rs",
				"location": {
					"file": "",
					"class": "org.jdbi.jdbi3.ResultSetRow",
					"method": "<init>",
					"line": 12
				},
				"stackTrace": [{
	"class": "org.jdbi.jdbi3.Handle$1",
	"method": "computeNext",
	"line": 45
},{
	"class": "org.jdbi.jdbi3.ResultSetRow",
	"method": "<init>",
	"line": 12
}]
			}
		]
	}
}

,

{
"type": "CONFLICT",
"label": "SVFA conflict",
"body": {
  "description": "SVFA conflict",
  "interference": [{
"type": "SourceNode",
"branch":"",
"text": "$stack9 := @caughtexception",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.Handle$1",
  "method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
  "line": "45"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)"]
}, {
"type": "SinkNode",
"branch":"",
"text": "e = $stack9",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.Handle$1",
  "method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
  "line": "53"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)"]
}]
}
},
{
"type": "CONFLICT",
"label": "SVFA conflict",
"body": {
  "description": "SVFA conflict",
  "interference": [{
"type": "SourceNode",
"branch":"",
"text": "$stack7 = new org.jdbi.jdbi3.ResultSetRow",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.Handle$1",
  "method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
  "line": "45"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack7.<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>($stack8), 45)"]
}, {
"type": "SimpleNode",
"branch":"",
"text": "this := @this: org.jdbi.jdbi3.ResultSetRow",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.ResultSetRow",
  "method": "<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>",
  "line": "-1"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack7.<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>($stack8), 45)"]
}, {
"type": "SinkNode",
"branch":"",
"text": "specialinvoke this.<java.lang.Object: void <init>()>()",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.ResultSetRow",
  "method": "<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>",
  "line": "10"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack7.<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>($stack8), 45)","(<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>, specialinvoke this.<java.lang.Object: void <init>()>(), 10)"]
}]
}
}

,

{
"type": "CONFLICT",
"label": "SVFA conflict",
"body": {
  "description": "SVFA conflict",
  "interference": [{
"type": "SourceNode",
"branch":"",
"text": "$stack11 := @caughtexception",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.Handle",
  "method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
  "line": "60"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)"]
}, {
"type": "SimpleNode",
"branch":"",
"text": "stmt#2 = $stack11",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.Handle",
  "method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
  "line": "62"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, stmt#2 = $stack11, 62)"]
}, {
"type": "SimpleNode",
"branch":"",
"text": "cause := @parameter0: java.lang.Throwable",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.JDBIException",
  "method": "<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>",
  "line": "-1"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)"]
}, {
"type": "SinkNode",
"branch":"",
"text": "specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause)",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.JDBIException",
  "method": "<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>",
  "line": "7"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)","(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)"]
}]
}
},
{
"type": "CONFLICT",
"label": "SVFA conflict",
"body": {
  "description": "SVFA conflict",
  "interference": [{
"type": "SourceNode",
"branch":"",
"text": "rs = interfaceinvoke stmt.<java.sql.PreparedStatement: java.sql.ResultSet executeQuery()>()",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.Handle",
  "method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
  "line": "35"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, rs = interfaceinvoke stmt.<java.sql.PreparedStatement: java.sql.ResultSet executeQuery()>(), 35)"]
}, {
"type": "SimpleNode",
"branch":"",
"text": "l2 := @parameter1: java.sql.ResultSet",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.Handle$1",
  "method": "<org.jdbi.jdbi3.Handle$1: void <init>(org.jdbi.jdbi3.Handle,java.sql.ResultSet)>",
  "line": "-1"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, specialinvoke tmp$79620878.<org.jdbi.jdbi3.Handle$1: void <init>(org.jdbi.jdbi3.Handle,java.sql.ResultSet)>(this, rs), 37)"]
}, {
"type": "SinkNode",
"branch":"",
"text": "this.<org.jdbi.jdbi3.Handle$1: java.sql.ResultSet val$rs> = l2",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.Handle$1",
  "method": "<org.jdbi.jdbi3.Handle$1: void <init>(org.jdbi.jdbi3.Handle,java.sql.ResultSet)>",
  "line": "38"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, specialinvoke tmp$79620878.<org.jdbi.jdbi3.Handle$1: void <init>(org.jdbi.jdbi3.Handle,java.sql.ResultSet)>(this, rs), 37)","(<org.jdbi.jdbi3.Handle$1: void <init>(org.jdbi.jdbi3.Handle,java.sql.ResultSet)>, this.<org.jdbi.jdbi3.Handle$1: java.sql.ResultSet val$rs> = l2, 38)"]
}]
}
},
{
"type": "CONFLICT",
"label": "SVFA conflict",
"body": {
  "description": "SVFA conflict",
  "interference": [{
"type": "SourceNode",
"branch":"",
"text": "e = $stack9",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.Handle$1",
  "method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
  "line": "53"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)"]
}, {
"type": "SinkNode",
"branch":"",
"text": "specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e)",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.Handle$1",
  "method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
  "line": "55"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)"]
}]
}
},
{
"type": "CONFLICT",
"label": "SVFA conflict",
"body": {
  "description": "SVFA conflict",
  "interference": [{
"type": "SourceNode",
"branch":"",
"text": "e = $stack9",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.Handle$1",
  "method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
  "line": "53"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)"]
}, {
"type": "SimpleNode",
"branch":"",
"text": "cause := @parameter0: java.lang.Throwable",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.JDBIException",
  "method": "<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>",
  "line": "-1"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)"]
}, {
"type": "SinkNode",
"branch":"",
"text": "specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause)",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.JDBIException",
  "method": "<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>",
  "line": "7"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)","(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)"]
}]
}
},
{
"type": "CONFLICT",
"label": "SVFA conflict",
"body": {
  "description": "SVFA conflict",
  "interference": [{
"type": "SourceNode",
"branch":"",
"text": "$stack7 = new org.jdbi.jdbi3.ResultSetRow",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.Handle$1",
  "method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
  "line": "45"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack7.<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>($stack8), 45)"]
}, {
"type": "SimpleNode",
"branch":"",
"text": "this := @this: org.jdbi.jdbi3.ResultSetRow",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.ResultSetRow",
  "method": "<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>",
  "line": "-1"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack7.<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>($stack8), 45)"]
}, {
"type": "SinkNode",
"branch":"",
"text": "specialinvoke this.<java.lang.Object: void <init>()>()",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.ResultSetRow",
  "method": "<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>",
  "line": "10"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack7.<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>($stack8), 45)","(<org.jdbi.jdbi3.ResultSetRow: void <init>(java.sql.ResultSet)>, specialinvoke this.<java.lang.Object: void <init>()>(), 10)"]
}]
}
},
{
"type": "CONFLICT",
"label": "SVFA conflict",
"body": {
  "description": "SVFA conflict",
  "interference": [{
"type": "SourceNode",
"branch":"",
"text": "$stack9 := @caughtexception",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.Handle$1",
  "method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
  "line": "45"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)"]
}, {
"type": "SourceNode",
"branch":"",
"text": "e = $stack9",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.Handle$1",
  "method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
  "line": "53"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)"]
}, {
"type": "SimpleNode",
"branch":"",
"text": "cause := @parameter0: java.lang.Throwable",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.JDBIException",
  "method": "<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>",
  "line": "-1"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)"]
}, {
"type": "SinkNode",
"branch":"",
"text": "specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause)",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.JDBIException",
  "method": "<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>",
  "line": "7"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)","(<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>, specialinvoke this.<java.lang.RuntimeException: void <init>(java.lang.Throwable)>(cause), 7)"]
}]
}
},
{
"type": "CONFLICT",
"label": "SVFA conflict",
"body": {
  "description": "SVFA conflict",
  "interference": [{
"type": "SourceNode",
"branch":"",
"text": "rs = interfaceinvoke stmt.<java.sql.PreparedStatement: java.sql.ResultSet executeQuery()>()",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.Handle",
  "method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
  "line": "35"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, rs = interfaceinvoke stmt.<java.sql.PreparedStatement: java.sql.ResultSet executeQuery()>(), 35)"]
}, {
"type": "SinkNode",
"branch":"",
"text": "specialinvoke tmp$79620878.<org.jdbi.jdbi3.Handle$1: void <init>(org.jdbi.jdbi3.Handle,java.sql.ResultSet)>(this, rs)",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.Handle",
  "method": "<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>",
  "line": "37"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle: java.util.stream.Stream query(java.lang.String)>, specialinvoke tmp$79620878.<org.jdbi.jdbi3.Handle$1: void <init>(org.jdbi.jdbi3.Handle,java.sql.ResultSet)>(this, rs), 37)"]
}]
}
},
{
"type": "CONFLICT",
"label": "SVFA conflict",
"body": {
  "description": "SVFA conflict",
  "interference": [{
"type": "SourceNode",
"branch":"",
"text": "$stack9 := @caughtexception",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.Handle$1",
  "method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
  "line": "45"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)"]
}, {
"type": "SourceNode",
"branch":"",
"text": "e = $stack9",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.Handle$1",
  "method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
  "line": "53"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, e = $stack9, 53)"]
}, {
"type": "SinkNode",
"branch":"",
"text": "specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e)",
"location": {
  "file": "",
  "class": "org.jdbi.jdbi3.Handle$1",
  "method": "<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>",
  "line": "55"
},
"stackTrace": ["(<org.jdbi.jdbi3.Handle$1: org.jdbi.jdbi3.ResultSetRow computeNext()>, specialinvoke $stack10.<org.jdbi.jdbi3.JDBIException: void <init>(java.lang.Throwable)>(e), 55)"]
}]
}
}

]
